lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -1982,4 +1982,17 @@ public void deleteLogForLessonsItem(SimplePageItem item) {
 			log.warn("Failed to delete lessons log for item {}", item.getId());
 		}
 	}
+
+  public void deleteQuestionResponsesForItem(SimplePageItem item) {
+		try {
+			DetachedCriteria d = DetachedCriteria.forClass(SimplePageQuestionResponse.class).add(Restrictions.eq("questionId",item.getId()));
+			DetachedCriteria d2 = DetachedCriteria.forClass(SimplePageQuestionResponseTotals.class).add(Restrictions.eq("questionId",item.getId()));
+			getHibernateTemplate().deleteAll(getHibernateTemplate().findByCriteria(d));
+			getHibernateTemplate().deleteAll(getHibernateTemplate().findByCriteria(d2));
+		} catch (DataAccessException e) {
+			log.warn("Failed to delete SimplePageQuestion responses for item {}", item.getId());
```

Comment:
I think this should be an error. It's a bad situation where we get this kind of exception. I'd also add e.toString() to the log error.

Reply:
Thank you for your input! I did update the PR

Reply:
Thank you for your input! I did update the PR